symplyphysics/laws/electricity/ampere_law.py

```diff
@@ -0,0 +1,51 @@
+from sympy import (Eq, solve, cos)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless
+)
+
+# Description
+## Ampere's law is the law that determines the force with which a magnetic field acts on
+## a small segment of a conductor with a current. The force turns out to be linearly dependent 
+## on both current and magnetic induction.
+
+## Law is: F = I * L * B * cos(a), where
+## F - force,
+## I - current,
+## L - length of conductor,
+## B - magnetic induction,
+## a - angle between magnetic induction and current direction.
+
+force = Symbol("force", units.force)
+
+current = Symbol("current", units.current)
+length = Symbol("length", units.length)
+induction = Symbol("induction", units.magnetic_density)
+angle = Symbol("angle", dimensionless)
```

Comment:
You can use `angle_type` here, I think.

Reply:
> You can use `angle_type` here, I think.

I can't find "angle_type" in the "units" module.

Reply:
It's in `symplyphysics`:
`from symplyphysics import angle_type`